modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java

```diff
@@ -627,6 +627,21 @@ public boolean isLimitToOneSubmissionPerUserEnabled()
 		return ddmFormInstanceSettings.limitToOneSubmissionPerUser();
 	}
 
+	public boolean isLoggedUser() {
+		ThemeDisplay themeDisplay = (ThemeDisplay)_renderRequest.getAttribute(
+			WebKeys.THEME_DISPLAY);
+
+		Layout layout = themeDisplay.getLayout();
+
+		if ((!layout.isPrivateLayout() && !themeDisplay.isSignedIn()) ||
+			isDefaultUser()) {
```

Comment:
Hi @marcelabc,
Why do we need this `isDefaultUser()`?

Reply:
I believe that the method isDefaultUser() is one way to know if the user is a guest. I will remove because themeDisplay.isSignedIn() is enough to identify if the user is logged.